test/src/unit-ordered-dim-label-reader.cc

```diff
@@ -403,16 +403,16 @@ TEST_CASE_METHOD(
   write_labels(1, 4, {1.0, 2.0, 3.0, 4.0});
   REQUIRE_THROWS_WITH(
```

Comment:
Though not part of this PR per se, in general, I would suggest using CHECK rather than REQUIRE.  The exception being if the test cannot proceed past an assertion without the assertion passing.